### ISSUE TITLE
simple-fields-field select won't open if padding is applied

### ISSUE THREAD

**Issue by btopro** [elmsln/issues#2002]
As part of the #2001 it was discovered that making the select box look nice via DDD leads to you not being able to activate the combo box directly to see it's options unless you click below it. We should be able to do some internal padding hacks to apply this (i think..) but this appears to be a web platform issue that can be tracked here: https://github.com/whatwg/html/issues/9757

Until we get the ability to open the picker directly we won't be able to solve this in a normal way.

**Comment by btopro** [comment]
leaving part behind and leveraging css for it as opposed to the outer element we get both needs met
